src/index.js

```diff
@@ -22,6 +22,7 @@ import add from './add'
 import subtract from './subtract'
 import divide from './divide'
 import multiply from './multiply'
+import object-to-arraylist from './object-to-arraylist'
```

Comment:
This is invalid JavaScript syntax (identifiers cannot have `-` in their name). You can change this to:

```javascript
import objectToArray from './object-to-arraylist'
```

Then, you'll also need to add this to the export at the bottom like the rest of the utility functions.